packages/core/src/App/AppContent.tsx

```diff
@@ -22,6 +24,11 @@ import Routes from './Containers/Routes/routes.jsx';
 import Devtools from './Devtools';
 import initDatadog from '../Utils/Datadog';
 
+const gb = new GrowthBook({
+    apiHost: 'https://cdn.growthbook.io',
+    clientKey: process.env.GROWTHBOOK_CLIENT_KEY,
+});
+
```

Comment:
this will cause `growthbook` to initialize 2 times in our app and there is one more caveat to this. we have kept the growthbook initialisation behind this `firebase` remote config, which is not happening now in this scenario.

Anyways, I think you should try doing `Analytics.initialise` in these lines. and it should be the same result, except you need to do one more change in the `useEffect` instead of doing `await gb.loadFeatures()` do this `await Analytics?.getInstances()?.ab?.GrowthBook?.loadFeature()`

Reply:
sure will try it,
but it does not return me a promise. no reason to await it..
thanks @amir-deriv

Reply:
@yauheni-deriv it's not the `init` function which is coming from `analytics` module. this one is same as `await gb.loadFeatures()`

Reply:
<img width="623" alt="image" src="https://github.com/binary-com/deriv-app/assets/129206554/555aa2ba-b0a3-4270-8387-df1448a2b3f7">
btw, as I checked. this is returning a promise actually

Reply:
I'll try again Analytics?.isFeatureOn

Reply:
after the latest discussions we will remove analytica and use firebase